source/reference/operator/update/addToSet.txt

```diff
@@ -33,12 +33,36 @@ elements in the modified set.
 If the field is absent in the document to update, :update:`$addToSet`
 creates the array field with the specified value as its element.
 
-If the field is **not** an array, the operation will fail.
+If the field is **not** an array, the operation will fail with an
+error message stating that :update:`$addToSet` cannot be applied to a
+non-array field.
```

Comment:
should either render the error message/response exactly here (which has pros and cons) or leave this more or less as it is.